### PR TITLE
 Positioning-and-Floating-Elements 

### PR DESCRIPTION
**Pull request by tiagoes**
Please kindly review our project

### CODE REVIEW

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Status: Changes required 🚨

## Hello @tiagoes and @GraceOyiza 🖐. Impressive project 👏🙌!

### However, we need to work on some changes before we can merge.

- Remember to add a descriptive PR, like, what you have done in this project, etc....
- There are webhint and stylelint errors. Remember to fix these errors before submitting for code review.

image

image

- The social icons and some texts are overlapping with the top navbar.

image

- The NY Times logo must be a link ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0))

- There is extra space below the footer.

#### Clone
image

#### Original
image

- The fixed footer is missing.

#### Clone
Annotation 2020-06-23 224234

#### Original
Annotation 2020-06-23 224307

- Your CSS file has one invalid property. Check the comments below.

File: styles/index.css [hunk 1]
```diff
@@ -0,0 +1,586 @@
...
+
+.aside-container-box h3 span {
+  font-weight: bold;
+}
+
+.subtext::before {
+  content: "";
+  display: block;
+  margin: 0 auto 20px auto;
+  width: 1 2.5%;
```
Comment (tirthajyoti-ghosh) [inline styles/index.css]: `width` only takes one value.

**Review by tirthajyoti-ghosh — COMMENTED** [review]
Your project is really amazing! You guys have done an incredible job. 
Before you submit any project for review be sure to fix all the stylelint errors.

Apply these changes and request another review.

Happy coding 👨‍💻

**Comment by tiagoes** [comment]
All the necessaries changes have been done.
Please review our code again.

**Comment by andresporras3423** [comment]
### **Pull Request Under Review By Oscar Russi (TSE)**

File: styles/index.css [hunk 2]
```diff
@@ -0,0 +1,640 @@
...
+
+.footer-main-column h3 {
+  font-size: 13px;
+  font-weight: 700;
+  color: rgb(51, 51, 51);
+}
+
+.aside-container-box h3:hover {
+  text-decoration: underline;
+}
```
Comment (andresporras3423) [inline styles/index.css]: This selector has the same style definition of selectors on lines 169 and 196. Group selectors to minimize your code.

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello @tiagoes @GraceOyiza** 
Good job with this new submission for this project :muscle:  
Your page really looks like a clone of the original :clap: :clap: :clap:  
Your project passes most of the requirements. There are just a few details to improve before you can go ahead:
- [ ] Don't repeat yourself, in order to minimize code you can group selectors with the same style definition (in your CSS files).

### OPTIONAL

- [ ] your footer tag should be outside your main tag and it should be the last element in your HTML

Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras

**Comment by tiagoes** [comment]
All the necessaries changes have been done.
Please review our code again.

**Review by christianotieno — APPROVED** [review]
# PROJECT APPROVED :champagne:
Excellent job on this project guys. :fireworks:  Keep it up.
##
### The [clone](https://raw.githack.com/tiagoes/Positioning-and-Floating-Elements/main-branch/index.html) shows good understanding and usage of the positioning and floating of elements. 💯 
##
- The cloned page is rendering as it should and meets all the project specifications.
-  The project has a descriptive README.md and is professionally done. Good job with that.
- Pull Request has a meaningful title and a short description.
- CSS is clean. No linter errors at this point and no bad practices used as well.
- The markup as well has no errors nor warnings to show. Good job with that.
- The repo has followed all the guidelines there are to follow (GitHub flow etc.)
**All the required tasks have been fulfilled in this project. I will go ahead and mark this as complete.**
- Congratulations on the splendid job @tiagoes  & @GraceOyiza! :+1:
- You can now merge these changes. 
- I wish you all the very best in your next tasks 🥇 
##
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)

**Pull request merged**